File: Rust/String/Excel-Sheet-Column-Title.rs
pub struct Solution {}

impl Solution {
    pub fn convert_to_title(mut column_number: i32) -> String {
        let mut result = String::new();
        while column_number > 0 {
            column_number -= 1;
            result.insert(0, (65 + (column_number % 26) as u8) as char);
            column_number /= 26;
        }

        result
    }
}
fn main() {}

File: Javascript/Excel-Sheet-Column-Title.js
/**
 * @param {number} columnNumber
 * @return {string}
 */
var convertToTitle = function (columnNumber) {
  let result = "";

  let base = 65; // "A" => 65

  while (columnNumber > 0) {
    columnNumber--;
    result = String.fromCharCode(base + (columnNumber % 26)) + result;
    columnNumber = Math.floor(columnNumber / 26);
  }

  return result;
};
